terraform/stacks/main/outputs.tf

```
@@ -396,12 +396,8 @@ output "elasticsearch_security_group" {
 }
 
 /* S3 Gateway Endpoint CIDRs for CF ASGs*/
-output "s3_gateway_endpoint_cidr_1" {
-  value = data.aws_prefix_list.s3_gw_cidrs.cidr_blocks[0]
-}
-
-output "s3_gateway_endpoint_cidr_2" {
-  value = data.aws_prefix_list.s3_gw_cidrs.cidr_blocks[1]
+output "s3_gateway_endpoint_cidrs" {
+  value = data.aws_prefix_list.s3_gw_cidrs.cidr_blocks[*]
```

```suggestion
  value = data.aws_prefix_list.s3_gw_cidrs.cidr_blocks
```

Upon further review, I don't even think we need the `[*]`. The `cidr_blocks` attribute is already a list of IP CIDRs: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/prefix_list#attribute-reference